x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/documents.tsx

```diff
@@ -141,11 +144,29 @@ export const SearchIndexDocuments: React.FC = () => {
           </EuiFlexGroup>
         </EuiFlexItem>
         <EuiFlexItem>
-          {docs.length === 0 &&
+          {isAccessControlIndexNotFound && (
+            <EuiCallOut
+              size="m"
+              title={i18n.translate(
+                'xpack.enterpriseSearch.content.searchIndex.documents.noIndex.title',
+                { defaultMessage: 'Access Control Index does not exist yet' }
+              )}
+              iconType="iInCircle"
+            >
+              <p>
+                {i18n.translate('xpack.enterpriseSearch.content.searchIndex.documents.noIndex', {
+                  defaultMessage:
+                    "Access Control Index doesn't exist yet. Access Control Index will be created for you when first Document Level Security sync is finished.",
```

Comment:
```suggestion
                    "An Access Control Index won't be created until you run your first access control sync, after enabling document-level security.",
```

@sphilipse how about this?